ansible/roles/schulcloud-server-core/templates/svc.yml.j2

```diff
@@ -12,6 +12,10 @@ spec:
       targetPort: 3030
       protocol: TCP
       name: api
+    - port: 4030
```

Comment:
I do not know if it is a good way to place it dirctly in the same file. It works but for example managment-svc.yml.j2 use also server module. (is only deployed on staging also maybe other case)
But i can not say finale is it the correct place from devops side. 
 @mamutmk5 can you look into it and say if it is ok for you?

Reply:
I don't know if it's needed to be honest as we moved to seperate deployment maybe it can be deleted

Reply:
So as I understood this can be removed right @mamutmk5?

Reply:
done